gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/DichotomyParallelization.java

```diff
@@ -19,25 +21,40 @@
 import com.farao_community.farao.swe.runner.app.services.CneFileExportService;
 import com.farao_community.farao.swe.runner.app.services.OutputService;
 import com.farao_community.farao.swe.runner.app.services.VoltageCheckService;
+import org.slf4j.Logger;
+import org.slf4j.MDC;
 import org.springframework.stereotype.Service;
 
+import java.util.Collection;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 /**
  * @author Theo Pascoli {@literal <theo.pascoli at rte-france.com>}
  */
 @Service
 public class DichotomyParallelization {
-
+    private final Logger businessLogger;
     private final DichotomyLogging dichotomyLogging;
     private final DichotomyRunner dichotomyRunner;
     private final OutputService outputService;
     private final VoltageCheckService voltageCheckService;
     private final CneFileExportService cneFileExportService;
+    public static final String SUMMARY = "Summary [{}] :  " +
```

Comment:
Could this be relocated to the dichotomyLogging class ? So this class does not perform anything related to building logs

Reply:
Yes I agree